src/typography/index.md

```diff
@@ -25,15 +25,17 @@ The Government Digital Service adapted it in 2012 for digital use, bringing the
 
 Using GDS Transport makes GOV.UK easier to recognise and information easier to read on any device.
 
-![TODO](./transport-font.svg)
+<div class="scale width-80 right edge">
+
+![Sample of GDS Transport font in heavy (bold) and regular weight](./transport-font.svg)
```

Comment:
Why do you use the word "heavy" when it is not used elsewhere in the text?
(Same for the other three instances below.)

Reply:
It's one typeface but two fonts. So, technically this sentence is incorrect. (If I understood that correctly?)
Same for the other one occurrence below.

Reply:
Note to self: Also fix TODOs in src/colour/govuk-blue/index.md